js/main.js

```diff
@@ -29,5 +32,97 @@ function submitForm(event) {
   data.entries.unshift(inputValues);
   data.nextEntryId++;
   newEntryForm.reset();
+
+  var newEntry = renderEntry(data.entries[0]);
+  ulElement.prepend(newEntry);
+
+  entriesView.className = 'view';
+  entryFormView.className = 'view hidden';
+  data.view = 'entries';
```

Comment:
I also suggest you do the work of assigning a value to `data.view` within the `switchView` function when you find the right view. That way you don't need to remember to set it yourself every time you switch views. That sort of scenario sets you up for lots of bugs.